docs/examples/subprocess.md

```diff
@@ -10,6 +10,7 @@
   [stdin](https://doc.deno.land/builtin/stable#Deno.stdin),
   [stdout](https://doc.deno.land/builtin/stable#Deno.stdout) and
   [stderr](https://doc.deno.land/builtin/stable#Deno.stderr) streams.
+- Use a specific shell by providing its path/name and it's string input switch, e.g. `Deno.run({cmd: ["bash", "-c",'"ls -la"']});
```

Comment:
```suggestion
- Use a specific shell by providing its path/name and its string input switch, e.g. `Deno.run({cmd: ["bash", "-c", '"ls -la"']});
```
The double quotes aren't necessary with bash, that's a cmd.exe quirk.

`-c` is correct for bash, `/d /s /c` is what you want for cmd.exe.